processor/resourcedetectionprocessor/README.md

```diff
@@ -93,11 +99,13 @@ ec2:
 ## Configuration
 
 ```yaml
-# a list of resource detectors to run, valid options are: "env", "system", "gce", "ec2", "ecs", "elastic_beanstalk", "azure"
+# a list of resource detectors to run, valid options are: "env", "system", "gce", "ec2", "ecs", "elastic_beanstalk", "eks", "azure"
 detectors: [ <string> ]
 # determines if existing resource attributes should be overridden or preserved, defaults to true
 override: <bool>
 ```
 
+Note that if multiple detectors are inserting the same attribute name, the first detector to insert wins. For example if you had `detectors: [eks, ec2]` then `cloud.infrastructure_service` will be `aws_eks` instead of `ec2`.
```

Comment:
Can we provide a recommended list for correct order in most use cases? X-Ray SDK uses elastic_beanstalk -> eks -> ecs -> ec2

Reply:
Can you add this list? I'm not really familiar with X-Ray, beanstalk, etc. I'm just focused on EKS for this change.

Reply:
Well - just need to copy my list below this line :)

Reply:
k, will add